src/components/WebexInterstitialMeeting/WebexInterstitialMeeting.scss

```diff
@@ -22,3 +22,9 @@
     flex-grow: 1;
   }
 }
+
+.interstitial-media-modal {
```

Comment:
if outside of the main container (`.#{$WEBEX_COMPONENTS_CLASS_PREFIX}-interstitial-meeting`), this needs to be prefixed as well. And maybe call it `{PREFIX}-interstitial-meeting__media-access-modal`

Reply:
or, better yet, just send a classname to the `<WebexMediaAccess meetingID={meetingID} />` component, with a `2rem` margin.